src/modules/sub-items/sub.items.module.js

```diff
@@ -334,5 +337,6 @@ SubItemsModule.defaultProps = {
         }
     },
     limit: 10,
-    offset: 0
+    offset: 0,
+    locationViewLink: '/admin/content/location/{{locationId}}'
```

Comment:
I think this should not be hardcoded here. If we want to hardcode it for now, it should be in the admin-ui with TODO for backend to provide this link